templates/partials/question.html

```diff
@@ -3,8 +3,12 @@
 {% from "components/panel/_macro.njk" import onsPanel %}
 
 {% set form = content.form %}
-
-{% set question_title = question.title %}
+{% set interviewer_note = _("Interviewer Note:") %}
+{% if question.interviewer_note %}
+  {% set question_title = '<mark class="instruction">' + interviewer_note + '</mark>' + question.title %}
+{% elif question.title %}
+  {% set question_title = question.title %}
```

Comment:
This should be formatted in a function, the same as the description and instruction are. I think then you may be able to reuse it on the interstitial partial.

Reply:
Is this ok to keep all this logic in frontend? I made a different variant of this change where the logic is in `question.py` and `content.py`..

Reply:
It sounds like this should actually really be a helper macro and we should be avoiding using jinja filters.

Reply:
I've added helper macro now, thanks!👍